src/helpers/hub.ts

```diff
@@ -28,7 +29,7 @@ export default class Hub {
 
     const listener = this.listeners[type];
     if (listener) {
-      listener(type, payload);
+      listener(payload);
```

Comment:
This is a huge breaking change. Why don't we reply to the person who raised the issue the object is on the second argument?

Reply:
We could,  but we did not solely make this changed based on a use request. The current implementation is wrong according to the specs: https://cdn.datacamp.com/dcl-react-prod/index.html#tutorial-customizing-integrating. 

So if we don't want to make this breaking change we have 2 options.
- Update the docs so they reflect reality, although it doesn't make any sense to me to pass the event as the first argument
- make the change but do it in a way that is backwards compatible for now and rectify this later when we release a new major version. We can easily check how many arguments are getting passed and jump into a different code based on that.

Reply:
I completely agree with you it doesn't make sense to pass the event type as the first argument. However, people using that feature probably found out the payload is passed on the second argument. This was suggested [here](https://github.com/datacamp/datacamp-light/issues/103#issuecomment-415328094) and looks like the user did it according to his [reply](https://github.com/datacamp/datacamp-light/issues/103#issuecomment-415418844).

With no versioning in place, I suggest keeping this fix for version 3.

Reply:
And as you suggested, update the documentation 😄